lib/index.js

```diff
@@ -39,18 +39,22 @@ const setGlobalOptions = function (options) {
 };
 
 const parseNamespacesFromEnvironment = function () {
```

Comment:
I am also not sure why this function is being called for *every* logger that is being created. Shouldn't it run only once, when the module first loads?